data/field_effect_scripts.s

```diff
@@ -10,66 +10,79 @@ gFieldEffectScriptPointers:: @ 82DB9D4
 	.4byte gFieldEffectScript_UseCutOnTree
 	.4byte gFieldEffectScript_Shadow
 	.4byte gFieldEffectScript_TallGrass
+
```

Comment:
Why did you put a newline after every five scripts?

Reply:
Oops! I meant to remove that. I had to find the 53rd entry or something, so I split it up to help counting.